lib/os-modules/Taskfile.yaml

```diff
@@ -7,13 +7,16 @@ vars:
     terraform-datadog-users \
     terraform-github-teams \
     terraform-github-organization \
+    terraform-googleworkspace-user-group-management \
+    terraform-googleworkspace-users-groups-automation \
```

Comment:
@westonplatter aren't we only using `-automation`? Can we archive or delete the fork of `-management`?

Reply:
Yes, we only need the `terraform-googleworkspace-users-groups-automation` repo. I still have an open PR on the `terraform-googleworkspace-user-group-management` repo so I'll keep it around, but it's not a module we depend on in `mp-infra`.

Reply:
Cool, got it. Removed!